chemprop/cli/train.py

```diff
@@ -451,6 +450,11 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
 
 
 def process_train_args(args: Namespace) -> Namespace:
+    if args.config_path is None and args.data_path is None:
+        raise ArgumentError(
+            argument=None, message="Must provide either a data path or a config path."
+        )
```

Comment:
Are there any cases in which the configuration file is provided but the `data_path` is not defined within it? This might only happen when users define the configuration file themselves. Do we want to provide the flexibility to allow users to generate the `config.json` themselves? I find the error message here a bit confusing; perhaps it should simply state that the `data_path` is missing.

Reply:
@shihchengli Thanks for the review! I updated the help string to make it more clear.